app/AppKernel.php

```diff
@@ -25,6 +25,7 @@ public function registerBundles()
             new Knp\Bundle\PaginatorBundle\KnpPaginatorBundle(),
             new Knp\Bundle\MenuBundle\KnpMenuBundle(),
             new Eo\AirbrakeBundle\EoAirbrakeBundle(),
+            new SimpleBus\SymfonyBridge\SimpleBusCommandBusBundle(),
```

Comment:
Do you need config for that, or is it just automatically when you add it in AppKernel?